examples/somersaultecu.py

```diff
@@ -364,6 +361,40 @@ class SID(IntEnum):
         compu_method=somersault_compumethods["boolean"]),
 }
 
+# tables
+somersault_tables = {
```

Comment:
Please feel free to change the content of this test table

Reply:
it is fine (quirky) :)